src/components/ButtonPanel.js

```diff
@@ -1,35 +1,36 @@
 import React from 'react';
+import Button from './Button';
 
 const ButtonPanel = () => (
   <div>
     <div className="group1">
-      <button type="button"> AC </button>
-      <button type="button"> +/-</button>
-      <button type="button">  % </button>
-      <button type="button">  ÷ </button>
+      <Button name="AC" color="rgb(240, 241, 243)" />
+      <Button name="+/-" color="rgb(240, 241, 243)" />
+      <Button name="%" color="rgb(240, 241, 243)" />
+      <Button name="÷" />
     </div>
     <div className="group1">
-      <button type="button"> 7 </button>
-      <button type="button"> 8 </button>
-      <button type="button"> 9 </button>
-      <button type="button"> X </button>
+      <Button name="7" color="rgb(240, 241, 243)" />
+      <Button name="8" color="rgb(240, 241, 243)" />
+      <Button name="9" color="rgb(240, 241, 243)" />
+      <Button name="X" />
     </div>
     <div className="group1">
-      <button type="button"> 4 </button>
-      <button type="button"> 5</button>
-      <button type="button"> 6</button>
-      <button type="button"> - </button>
+      <Button name="4" color="rgb(240, 241, 243)" />
+      <Button name="5" color="rgb(240, 241, 243)" />
+      <Button name="6" color="rgb(240, 241, 243)" />
+      <Button name="-" />
     </div>
     <div className="group1">
-      <button type="button"> 1 </button>
-      <button type="button"> 2</button>
-      <button type="button"> 3</button>
-      <button type="button"> + </button>
+      <Button name="1" color="rgb(240, 241, 243)" />
+      <Button name="2" color="rgb(240, 241, 243)" />
+      <Button name="3" color="rgb(240, 241, 243)" />
+      <Button name="+" />
     </div>
     <div className="group1">
-      <button type="button"> 0 </button>
-      <button type="button"> . </button>
-      <button type="button"> = </button>
+      <Button name="0" wide="true" color="rgb(240, 241, 243)" />
```

Comment:
This is causing a console error, see below:

![image](https://user-images.githubusercontent.com/36057474/74816652-d8747080-52fb-11ea-8857-b892efd57c4e.png)

For `props` with boolean values, let's omit the value if `true`.

```suggestion
      <Button name="0" wide color="rgb(240, 241, 243)" />
```